### ISSUE TITLE
Feature request: Visual indicators of assignment state in "Students and Their Results" table

### ISSUE THREAD

**Issue by JanAlster** [ReCodEx/web-app#396]
The "Students and Their Results" table on group's main page ("Group assignments and student results") provides a nice overview of student's achieved points for the supervisor. Clicking the points number for given student and assignment gives a floating window with detailed overview of all submitted solutions.

However it would be great if there was an icon indicator (or indicators) beside the points number marking that attention is needed for this particular student and assignment. In particular that 
- there is a solution that has not been reviewed yet
- none of the solutions was accepted as the final
- there is a new message in any of the solutions chat/comments
In short, a way to know which of the numbers in the table the supervisor should click to get where he/she/it needs to be.

**Comment by krulis-martin** [comment]
Accepted solution icon is definitely planned. Whether last submitted solution needs review might be also possible.
The "new message" indicator is not possible at the moment, since there is no way how to distinguish "new" messages.

**Comment by krulis-martin** [comment]
The accepted indicator was added to the table (will be in the next release).

**Comment by krulis-martin** [comment]
- Ad "there is a solution that has not been reviewed yet" - this is now handled by request for review indicators
- Accepted indicator was added long time ago.
- Ad "there is a new message in any of the solutions chat/comments" - I am sorry, but this will probably never be implemented. To do this properly, it will require to add a timestamp of last visit for every (discussion-thread, user) and then scan this structure for all solutions within a table separately. I didn't do any exact measurements, but we are struggling desperately to decrease the loading time of the "Students and Their Results" table (which is quite high for courses with many students and many assignments), and this will worsen the situation further.

I am closing this issue.
